index.html

```diff
@@ -36,16 +36,21 @@
         <div class="container home_2col">
             <div class="row">
                 <div class="col">
-                    <img src="images/house.jpg" class="rounded d-block dibley" alt="...">
+                    <img src="images/Vegas.jpg" class="rounded d-block dibley" alt="...">
                 </div>
                 <div class="col">
                     <Claudette Rankine: WEBD-225>This morning, shortly after 11:00, comedy struck this little house on Dibley Road.</h1>
                 </div>
             </div>
         </div><!--end 2col-->
 
-        <p>This is filler text, otherwise known as lorem ipsum. It is nonsense text to fill space. There are generators of funny lorem ipsum, such as the one from <a href="https://montypythonipsum.megert.se/">Monty Python</a> or <a href="http://fillerama.io/">others</a>. Oh, I don't have time for this. I have to go and buy a single piece of fruit with a coupon and then return it, making people wait behind me while I complain.</p>
-        <p>Perhaps your civilization is merely the sewer of an even greater society above you! If you do too much, people get dependent on you, and if you do nothing, they lose hope. You have to use a light touch. Like a safecracker, or a pickpocket. Do a flip!  It's toe-tappingly tragic! Too much work. Let's burn it and say we dumped it in the sewer. With a warning label this big, you know they gotta be fun!</p>
+        <p>This photo of the Sphere at the Venetian in Las Vegas was taken in April, 2024 when my husband and I were there celebrating our 34th anniversary. The Sphere is the world's largest spherical structure and architectural wonder.</p>
+        <p>Perhaps your civilization is merely the sewer of an even greater society above you! If you do too much, people get dependent on you, and if you do nothing, they lose hope.</p>
+       <article>Week 2</article>
```

Comment:
Week 2 needs to be nested n the h2, which itself needs to be inside the article.